pippo-core/src/main/java/ro/pippo/core/PippoConstants.java

```diff
@@ -67,6 +73,45 @@
 
     public static final String UTF8 = StandardCharsets.UTF_8.toString();
 
+    public static final String PIPPO_LOGO = "\n"
+        + " ____  ____  ____  ____  _____\n"
+        + "(  _ \\(_  _)(  _ \\(  _ \\(  _  )\n"
+        + " ) __/ _)(_  ) __/ ) __/ )(_)(   http://pippo.ro\n"
+        + "(__)  (____)(__)  (__)  (_____)  {}\n";
+
+    public static String getPippoLogo() {
```

Comment:
I propose you to move this method in a new class `PippoUtils` - for example `PippoUtils.getLogo()`, `PippoUtils.getVersion()` - and to keep in `PippoCostants` only constants.